docs/source/posts/2023/2023-06-03-week-1-praneeth.rst

```diff
@@ -0,0 +1,41 @@
+Week 1: Working with SpinBox and TextBox Enhancements
+=====================================================
+
+.. post:: June 3, 2023
+   :author: Praneeth Shetty
+   :tags: google
+   :category: gsoc
+
+What did you do this week?
+--------------------------
+This week, my focus was on reviewing pull requests (PRs) and issues related to the user interface (UI) of the project. I meticulously went through each PR and issue, identifying those specifically associated with UI improvements. To streamline the process, I categorized them accordingly under the UI category. One of the key tasks was PR `#499 <https://github.com/fury-gl/fury/pull/499>`__, which involved the implementation of SpinBoxUI. After rebasing the PR, I identified an alignment issue with the textbox component.
+
+To resolve this issue, I started by investigating the root cause. I discovered that the alignment was initially based on the position of the parent UI, which caused the text to extend beyond the boundaries of the textbox. To rectify this, I devised a solution where I calculated the background size of the textbox and adjusted the text's position accordingly. By aligning the text with the calculated size, I ensured a proper and visually appealing alignment within the textbox.
+
+To provide a clear understanding of the improvement, I have prepared a comparison of the textbox alignment before and after the modifications:
```

Comment:
Here, I would make `modifications:` -> `modifications.`, as below you already have `Before:`, which already does that job.